tests/constraints/test_constraints_continuous.py

```diff
@@ -68,6 +68,81 @@ def test_inequality3(campaign, n_iterations, batch_size):
     assert (1.0 * res["Conti_finite1"] + 3.0 * res["Conti_finite2"]).le(0.301).all()
 
 
+@pytest.mark.parametrize("parameter_names", [["Conti_finite1", "Conti_finite2"]])
+@pytest.mark.parametrize("constraint_names", [["InterConstraint_1"]])
+@pytest.mark.parametrize("batch_size", [5], ids=["b5"])
```

Comment:
I'll just remove the batch_sizes for the test. Would prefer to keep different numbers of iterations since it previously happened that it seemed like constraints were obeyed, while this was just a numerical coincidence, and this is at least a bit improved when doing more iterations. Please resolve if this is ok.

Reply:
ok